SingularityUI/app/controllers/TaskDetail.coffee

```diff
@@ -123,6 +132,38 @@ class TaskDetailController extends Controller
                 app.caughtError()
                 delete @models.resourceUsage
 
+    getAlerts: (requestTaskHistory) =>
+        task = @models.task
+        alerts = []
+        # Is this a scheduled task that has been running much longer than previous ones?
```

Comment:
instead of calculating this yourself, you should use `averageRuntimeMillis` in the [SingularityDeployStatistics](https://github.com/HubSpot/Singularity/blob/master/SingularityBase/src/main/java/com/hubspot/singularity/SingularityDeployStatistics.java#L27) object, which can be grabbed from the deploy history endpoint (`.../api/history/request/REQUEST_ID/deploy/DEPLOY_ID`). it would also be wise to surface `warnIfScheduledJobIsRunningPastNextRunPct` from https://github.com/HubSpot/Singularity/blob/master/SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java#L172 into the window.config object so that we don't have to hardcode the threshold value